src/SFA.DAS.Payments.MatchedLearner.Application/Mappers/MatchedLearnerDtoMapper.cs

```diff
@@ -24,14 +24,19 @@ public MatchedLearnerDto Map(MatchedLearnerDataLockInfo matchedLearnerDataLockIn
                 .ThenBy(x => x.LearningStartDate)
                 .First();
 
+            var firstSubmissionJob = matchedLearnerDataLockInfo.LatestSuccessfulJobs
+                .OrderByDescending(x => x.AcademicYear)
+                .ThenBy(x => x.CollectionPeriod)
+                .First();
+
             return new MatchedLearnerDto
             {
                 StartDate = firstEvent.LearningStartDate.GetValueOrDefault(),
                 EventTime = firstEvent.EventTime,
-                IlrSubmissionDate = firstEvent.IlrSubmissionDateTime,
-                IlrSubmissionWindowPeriod = firstEvent.CollectionPeriod,
-                AcademicYear = firstEvent.AcademicYear,
-                Ukprn = firstEvent.Ukprn,
+                IlrSubmissionDate = firstSubmissionJob.IlrSubmissionTime,
+                IlrSubmissionWindowPeriod = firstSubmissionJob.CollectionPeriod,
+                AcademicYear = firstSubmissionJob.AcademicYear,
+                Ukprn = firstSubmissionJob.Ukprn,
```

Comment:
Requirement is definitely that all 4 of these fields should be mapped from this job? Ticket only mentions CollectionPeriod & AcademicYear. If this is the case then consider this PR approved.